zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -56,7 +57,10 @@ func (ob *Observer) WatchInbound(ctx context.Context) error {
 			}
 			err := ob.ObserveInbound(ctx)
 			if err != nil {
-				ob.logger.Inbound.Error().Err(err).Msg("WatchInbound error observing in tx")
+				// skip showing log for block number 0 as it means Bitcoin is not being observed
+				if !strings.Contains(err.Error(), "current block number 0 is too low") {
```

Comment:
Where does this message come from? Binding logic to error's substring is brittle. Can we introduce a sentinel error for this? e.g. `ErrUnsupported...`

Reply:
93a7f830f